config/kubernetes-sigs/sig-network/teams.yaml

```diff
@@ -114,10 +115,9 @@ teams:
   ingress2gateway-maintainers:
     description: Write access to the ingress2gateway repo
     members:
-    - bowei
     - LiorLieberman
+    - mlavacca
     - robscott
-    - shaneutt
```

Comment:
You've _removed_ previous maintainers, what we do instead is we'll add a section called `emeritus` which doesn't carry any official position but denotes previously involved maintainers.